pvr.waipu: update 21.9.2-Omega to 21.9.3-Omega

heitbaum opened the pull request.

flubshi commented:
Thanks @heitbaum for the fast and regular updates of the addons! Are doing this manually or is it automated?

I noticed that the PR for pvr.waipu Omega release was created (this PR), but there is no PR for the Piers release of pvr.waipu.

heitbaum commented:
> Thanks @heitbaum for the fast and regular updates of the addons! Are doing this manually or is it automated?
> 
> I noticed that the PR for pvr.waipu Omega release was created (this PR), but there is no PR for the Piers release of pvr.waipu.

I have a hacky script I run. (For both 12 and 13) 
I then edit the commit message and push it.
I ran them this morning when I got up… only Omega was there at that time. I still do it manually, just to validate.

```
PROJECT=Generic ARCH=x86_64 DEVICE=Generic-legacy tools/mkpkg/update_binary-addons -x Omega | tee yy-output
echo "kodi-binary-addons: update to latest versions" > yy-output.updated
echo >> yy-output.updated
cat yy-output | grep "^UPDATED " | sed 's/^UPDATED /- /' | sed 's/ from /: update /' >> yy-output.updated

git co -- packages/mediacenter/kodi-binary-addons/audiodecoder.fluidsynth/package.mk
git co -- packages/mediacenter/kodi-binary-addons/visualization.pictureit/package.mk

git commit -eF yy-output.updated .

git co -- packages/mediacenter/kodi-binary-addons/audiodecoder.fluidsynth/package.mk 
git co -- packages/mediacenter/kodi-binary-addons/visualization.pictureit/package.mk
```
